src/main/java/org/jenkinsci/plugins/workflow/steps/ExceptionCause.java

```diff
@@ -4,14 +4,7 @@
 
 import java.io.Serializable;
 
-/**
- * {@link CauseOfInterruption} that captures random {@link Throwable},
- * which is used when the cancellation is in response to some failures.
- *
- * TODO: move this to core
- * TODO: better summary.jelly
- * @author Kohsuke Kawaguchi
- */
+@Deprecated
```

Comment:
This is still used in `(Abstract)SynchronousStepExecution.stop`, which I think does not have any alternative, so maybe it shouldn't be deprecated.